hive/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -75,21 +77,23 @@ public TableMetadata refresh() {
       String tableType = table.getParameters().get(TABLE_TYPE_PROP);
 
       if (tableType == null || !tableType.equalsIgnoreCase(ICEBERG_TABLE_TYPE_VALUE)) {
-        throw new IllegalArgumentException(format("Invalid tableName, not Iceberg: %s.%s", database, table));
+        throw new IllegalArgumentException(String.format("Invalid tableName, not Iceberg: %s.%s", database, table));
```

Comment:
Would it make sense to simply use `Preconditions.checkArgument` here [and other places where it makes sense] which supports formatting out of the box?

Reply:
+1 to this